packages/block-library/src/container/style.scss

```diff
@@ -0,0 +1,5 @@
+.wp-block-container {
+	// 1px top/bottom padding allows us to prevent margin collapsing while
```

Comment:
Does margin collapse happen on the front-end, or is this meant to offset the editor styles? If the latter, should we then include this only in an editor-specific stylesheet?

Reply:
Front and back-end, but in my testing it was good for both cases.